src/cmds/docs/single.js

```diff
@@ -8,12 +8,12 @@ const pushDoc = require('../../lib/pushDoc');
 module.exports = class SingleDocCommand {
   constructor() {
     this.command = 'docs:single';
-    this.usage = 'docs:single <filepath> [options]';
+    this.usage = 'docs:single <file> [options]';
```

Comment:
Doesn't `<file>` need to match the arg that we're going to be using (`filePath` in this case)?

Reply:
Nope, this is just the string used for the usage example

Reply:
It's been so long since I've looked at this part of the code but how does it know to give you the first arg as `filePath`?

Reply:
I swear I thought it was parsing the usage string to determine if things were required or not

Reply:
this guy: https://github.com/readmeio/rdme/blob/aaad56d984da47ed3576a8b672de5b848a07ff8d/src/cmds/docs/single.js#L28-L32

Reply:
Ah there it is. Ignore me today!